code/modules/hydroponics/grown/ambrosia.dm

```diff
@@ -49,3 +49,25 @@
 	icon_state = "ambrosiadeus"
 	filling_color = "#008B8B"
 	origin_tech = "biotech=4;materials=3"
+
+//Ambrosia Gaia
+/obj/item/seeds/ambrosia/gaia
+	name = "pack of ambrosia gaia seeds"
+	desc = "These seeds grow into ambrosia gaia, incredibly rare but imbued with the power of a whole ecosystem."
+	icon_state = "seed-ambrosia_gaia"
+	species = "ambrosia_gaia"
+	plantname = "Ambrosia Gaia"
+	product = /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/gaia
+	mutatelist = list()
+	reagents_add = list("earthsblood" = 0.4)
```

Comment:
i'm gonna be honest here adminord is pretty tame nowadays compared to a competent chemist mix